switch_config_render/generate_svg.py

```diff
@@ -56,9 +57,7 @@ def render_box(self, canvas, x, y):
             box.add(
                 canvas.drawing.text(
                     self.name,
-                    insert=(x + self.width / 2.0, y + self.height - 20),
-                    alignment_baseline="middle",
-                    text_anchor="middle",
+                    insert=(x + 20, y + self.height - 20),
                     fill="black",
                     style="font-family:Courier New",
```

Comment:
Not every host is going to have the Courier New typeface (e.g. macs or linux machines).
```suggestion
                    style="font-family: monospace",
```